test_voc.ino

```diff
@@ -1,17 +1,11 @@
 #if false
 
-void setup() {
-  Serial.begin(9600);
-  while (!Serial) { /* Wait for Serial monitor to connect */ }
-  Serial.println("Connecting to sensor...");
-}
```

Comment:
This function is still necessary. I was just saying that I don't think the print statement made sense.